site/include/header.php

```diff
@@ -1,50 +1,69 @@
 <?php
 
-function PrintHeader(){
-	echo'<div class="HeaderImage">
-		<img src="Curtain.jpg" alt="Curtains"
-		width="100%" height="30%" />
-		<div class="HeaderLine">
-			<h1>
-				St. Louis Theatre
-			</h1>
-		</div>
-	</div>
-	<div class="NavBar">
-		<a href="index.php">
-			<div class="NavButton">
-				<p>
-					Home
-				</p>
-			</div>
-		</a>
-		<a href="News.php">
-			<div class="NavButton">
-				<p>
-					News
-				</p>
-			</div>
-		</a>
-		<a href="Calendar.php">
-			<div class="NavButton">
-				<p>
-					Calendar
-				</p>
+function PrintHeader($SiteTitle, $PageTitle){
+	echo'
+	<head>
+	<link rel="stylesheet" type="text/css" href="Style.css">
+	<link href="https://fonts.googleapis.com/css?family=Raleway"
+	rel="stylesheet">
+		<title>'
```

Comment:
There are some nifty tricks you can use in PHP to make it easier to put variables into strings. what you're doing here is fine, but it's worth learning the tricks, so remind me next time we meet and I can explain some of them.